eppy/bunch_subclass.py

```diff
@@ -200,6 +224,16 @@ def fieldvalues(self):
         """
         return self.obj
 
+    @property
+    def functions(self):
+        try:
```

Comment:
This new EpBunch property is mainly used in the `__dir__` method to filter and list all possible functions for this type of EpBunch object. See the condition on line 232: self.key must be in the possible keys for this function (values[1]).

Here `value` is a 2-tuple: the first element is the function and the second element is the list of keys this function works with.